packages/itmat-job-executor/src/curation/JSONCurator.ts

```diff
@@ -0,0 +1,121 @@
+import csvparse from 'csv-parse';
+import { Collection } from 'mongodb';
+import { IJobEntry } from 'itmat-commons/dist/models/job';
+import { Writable } from 'stream';
+const fs = require('fs');
+const JSONStream = require('JSONStream');
+
+export interface IDataEntry {
+    m_jobId: string; // the id of the job that causes this upload
+    m_eid: string; // patient Id
+    m_study: string; // study Id
+    m_versionId: string; // data version Id 
+    [field: string]: {
+        [instance: string]: {
+            [array: number]: number | string
+        }
+    } | string | boolean | string[];
+}
+
+export interface IFieldDescriptionObject {
+    fieldId: number;
+    timepoint: number;
+    measurement: number;
+    datatype: 'c' | 'i' | 'd' | 'b' | 't';
+}
```

Comment:
Those classes are already defined in the `CSVCurator.ts` file. It would be better to have them abstracted out.